pixl_core/tests/conftest.py

```diff
@@ -13,6 +13,157 @@
 #  limitations under the License.
 from __future__ import annotations
 
+import datetime
 import os
+import shutil
+import subprocess
+from pathlib import Path
 
-os.environ["ENV"] = "test"
+import pytest
+from core.database import Base, Extract, Image
+from sqlalchemy import Engine, create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+os.environ["RABBITMQ_USERNAME"] = "guest"
+os.environ["RABBITMQ_PASSWORD"] = "guest"  # noqa: S105 Hardcoding password
```

Comment:
might be worth just adding the `S105` exception into the top level ruff exclusions for anything `**tests**`